materials/sql_joins.md

```diff
@@ -7,44 +7,65 @@ language: SQL
 
 ### Basic join
 
+* [`JOIN`](http://www.w3schools.com/sql/sql_join.asp) 
+    * combine rows from multiple tables
+    * based on condition
+* This query selects `species` records in the `PortalMammals_main` table and 
+  `scientificname` found in the `PortalMammals_species` table.
+    * The query links the `species` with `new_code` from `PortalMammals_species`.
```

Comment:
Some of these names don't match the current values in the Portal Project Teaching Database, but I've added a separate issue for that: #441.